crates/e2e-testing/src/spin.rs

```diff
@@ -117,5 +117,17 @@ pub fn registry_login(registry_url: &str, username: &str, password: &str) -> Res
 pub fn version() -> Result<String> {
     let output = utils::run(&["spin", "--version"], None, None)?;
     utils::assert_success(&output);
-    Ok(format!("{:#?}", std::str::from_utf8(&output.stdout)?))
+    Ok(format!(
+        "{:#?}",
+        std::str::from_utf8(&output.stdout)?.trim().to_string()
```

Comment:
I think you want this?
```suggestion
        "{}",
        std::str::from_utf8(&output.stdout)?.trim()
```

Reply:
done